00-get-started/01-ingest-kafka-data.md

```diff
@@ -40,15 +40,25 @@ bin/kafka-console-consumer.sh --topic test --from-beginning --bootstrap-server l
 
 Ensure that you have RisingWave up and running.
 
+```terminal
+./risingwave
+```
+
+In another terminal, connect to your RisingWave.
+
+```terminal
+psql -h localhost -p 4566 -d dev -U root
+```
+
 ### Create a source
 
-To connect to the data stream we just created in Kafka, we need to create a source using the `CREATE SOURCE` or `CREATE TABLE` command. Once this connection is established, RisingWave will be able to read any new messages from Kafka in real time. 
+To connect to the data stream we just created in Kafka, we need to create a source using the `CREATE SOURCE` or `CREATE TABLE` command. Once the connection is established, RisingWave will be able to read any new messages from Kafka in real time. 
 
-The following SQL query creates a source `website_visits_stream`. We also define a schema to map fields from the JSON data to the streaming data. 
+The following SQL query creates a source named `website_visits_stream`. We also define a schema here to map fields from the JSON data to the streaming data. 
 
 ```sql
 CREATE source IF NOT EXISTS website_visits_stream (
- timestamp TIMESTAMP,
+ timestamp timestamptz,
```

Comment:
Fix to use the correct data [type](https://docs.risingwave.com/docs/current/sql-data-types/): `timestampz` for timestamps with timezones.